src/examples/java/com/amazonaws/crypto/examples/FileStreamingDefaults.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except
+ * in compliance with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.amazonaws.crypto.examples;
+
+import com.amazonaws.encryptionsdk.AwsCrypto;
+import com.amazonaws.encryptionsdk.AwsCryptoInputStream;
+import com.amazonaws.encryptionsdk.AwsCryptoResult;
+import com.amazonaws.encryptionsdk.CreateDecryptingInputStreamRequest;
+import com.amazonaws.encryptionsdk.CreateEncryptingInputStreamRequest;
+import com.amazonaws.encryptionsdk.keyrings.Keyring;
+import com.amazonaws.encryptionsdk.keyrings.StandardKeyrings;
+import com.amazonaws.encryptionsdk.kms.AwsKmsCmkId;
+import com.amazonaws.util.IOUtils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * This example shows how to use the streaming encrypt and decrypt APIs when working with files.
+ * <p>
+ * One benefit of using the streaming API is that
+ * we can check the encryption context before we start decrypting.
+ * <p>
+ * In this example, we use an AWS KMS customer master key (CMK),
+ * but you can use other key management options with the AWS Encryption SDK.
+ * For examples that demonstrate how to use other key management configurations,
+ * see the 'keyring' and 'masterkeyprovider' directories.
+ */
+public class FileStreamingDefaults {
+
+    /**
+     * Demonstrate an encrypt/decrypt cycle using the streaming encrypt/decrypt APIs with files.
+     *
+     * @param awsKmsCmk           The ARN of an AWS KMS CMK that protects data keys
+     * @param sourcePlaintextFile Plaintext file to encrypt
+     */
+    public static void run(final AwsKmsCmkId awsKmsCmk, final File sourcePlaintextFile) throws IOException {
+        // Instantiate the SDK
```

Comment:
```suggestion
        // Instantiate the ESDK
```

Reply:
@juneb advised that we generally don't use the term ESDK. I'll just say Instantiate the AWS Encryption SDK, not really a need to abbreviate things here